md/product-versioning.md

```diff
@@ -0,0 +1,51 @@
+# Bonita Versioning
+Discover how Bonita manages its product and artifacts versions and how it can impact your projects
+
+## Product Versions
+
+Bonita follows the usual semantic versioning X.Y.z:
+* X is the major version number. It changes when non-backward compatible changes are implemented.
+* Y is the minor version number. It version changes when backward compatible features are added. A minor version is released every six months to improve continuously Bonita.
+* z is the maintenance version number. The maintenance versions contain fixes. Maintenance versions are released every month between two minor versions for the current minor version. These versions are cross-compatible by default for the same minor version.
```

Comment:
Why do we have a lowercase z? Seems like a typo.
I would add an example to make the things very clear: not sure "cross-compatible" is obvious for everybody.